pex/commands/command.py

```diff
@@ -281,6 +281,13 @@ def global_environment(options):
     :yields: The configured global environment.
     :raises: :class:`GlobalConfigurationError` if invalid global option values were specified.
     """
+    if not hasattr(options, "rc_file"):
```

Comment:
It would probably be a bit nicer to use `command_type` as the trigger, but this definitely works.

Reply:
You're right! Changed.

Reply:
Nope, turns out we can't use command_type because this code path is also used by the legacy command line parsing in pex/bin/pex.py, and that doesn't set command_type.